app/views/components/button_link.scala.html

```diff
@@ -17,5 +17,5 @@
 @(messageKey: String = "", href: String)(implicit messages: Messages)
 
 <div class="section">
-    <a href="@href" role="button" class="button">@messages(messageKey)</a>
+    <a href="@href" role="button" class="button" id="link-back">@messages(messageKey)</a>
```

Comment:
Why have you added an id here?  Is this not a general "button as a link" component?  The back_link component uses the id "back-link".

Reply:
This is some kind of `button-link` it's different than other back links we have on pages. I added ID to have a possibility to fetch this element on the page
what is your proposal about it?

Reply:
I agree with Steve, this is a generic `button` component

Pass in an ID as an argument if you need it

Reply:
My point was that the id seems to indicate this component has a specific use (to render a back link).   But I think the component is more general than that.  Maybe pass the id in as a parameter (defaulted to "button-link"? for views that don't care)?

Reply:
Yea - this is generic component should have id parameter or does not have it at all.

Reply:
Changed